docker-compose/technical/docker-compose.technical.yml

```diff
@@ -157,6 +157,18 @@ services:
       - $PWD/../technical/servers_pgadmin.json:/pgadmin4/servers.json
     restart: unless-stopped
 
+  minio:
```

Comment:
make it optional to be used by the people trying the s3 mode in docker compose

Reply:
Started in 'all' and 's3' profiles